CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 All notable changes to this project will be documented in this file.
 This project uses the changelog in accordance with [keepchangelog](http://keepachangelog.com/). Please use this to write notable changes, which is not the same as git commit log...
 
+## [Jrjgjk, Zolorah nexwatch improvement][2021-06-22]
+### Added
+ - Standalone mode for T55 (nexwatch) tag, dumping important values in the memory file. (@jrjgjk, @zolorah)
+ - Launch a brutefoce in lf nexwatch reader to find the magic number if not already known. (@jrjgjk, @zolorah)
+ - Can clone with an unknown magic byte in psk1 or psk2 modulation. (@jrjgjk, @zolorah)
+
```

Comment:
We add comment about changes,  will single line per addition.    And it should be in the unreleased section.
Look at how the current lines are styled,  and adapt your line accordingly